ovs/v1/ovs.yaml

```diff
@@ -23,28 +23,10 @@ paths:
       summary: Find schedules
       description: Returns all schedules filtered by the parameters.
       parameters:
-        - in: query
-          name: startDate
-          description: The start date of the period for which schedule information is requested. The value is populated in ISO 8601 Date format. If not provided, the current date is used by default.
-          required: false
-          schema:
-            $ref: '#/components/schemas/startDate'
-        - in: query
-          name: dateRange
-          description: The time period for which schedule information is sent. The duration is populated in ISO 8601 Duration format.
-          required: false
-          schema:
-            $ref: '#/components/schemas/dateRange'
-        - in: query
-          name: limit
-          description: Maximum number of items to return. 
-          schema:
-            $ref: '#/components/parameters/limitParam'
-        - in: query
-          name: cursor
-          description: A server generated value to specify a specific point in a collection result, used for pagination.
-          schema:
-            $ref: '#/components/parameters/cursorParam'        
+        - $ref: '#/components/parameters/startDate'
+        - $ref: '#/components/parameters/dateRange'
+        - $ref: 'https://api.swaggerhub.com/domains/dcsaorg/DCSA_OVS_DOMAIN/1.0.0#/components/parameters/limitParam'
+        - $ref: 'https://api.swaggerhub.com/domains/dcsaorg/DCSA_OVS_DOMAIN/1.0.0#/components/parameters/cursorParam'
```

Comment:
```suggestion
        - $ref: 'https://api.swaggerhub.com/domains/dcsaorg/DCSA_DOMAIN/1.0.0#/components/parameters/limitQueryParam'
        - $ref: 'https://api.swaggerhub.com/domains/dcsaorg/DCSA_DOMAIN/1.0.0#/components/parameters/cursorQueryParam' 
        - ```

Reply:
For limit and cursor we should use the globally defined parameters in DCSA_DOMAIN